### DISCUSSION TITLE
Release schedule/policy?

### DISCUSSION THREAD

**Discussion by rolandcrosby-dbt** [golangci/golangci-lint#3419] [Q&A] [answered]
Hi, I'm running into the issue documented in #3087. This appears to have been fixed in early November, but the latest release for this project is from October 22, so anyone installing the project's latest release (e.g. from Homebrew) will run into this bug (#3402, #3375). I don't see anything official in the documentation or the repo indicating when releases are triggered for this project; is there a regular release cadence? If not, could I request that a release be cut that includes the fix for #3087?

**Comment by ldez** [comment, marked as answer]
We will merge some current PRs and after we will create a release.

About the release, we have a lack of documentation, I'm working on that in the background.
